IDEOorg/steps-bot
# #163801895 If a user responds to the `helpcoachresponse` script, their new question text should be appended to the original Request.bg

#### What does this PR do?
This PR fixes the issue of creating a new request every time a request is sent. It creates a new request only when a client is sending a new message.
#### Description of Task to be completed?
- modify requestResolved variable on the script
- add logic to handle requests
- add a method to get a single request from a user
#### How should this be manually tested?
- clone the repo
- checkout to the branch `bg-append-response-to-original-request-163801895`
- send a help request
- login as a coach and reply to the help message
- reply the message from `twilio-gui`
- visit the coach app to see the new message updated
#### Any background context you want to provide?
When a client replies to a coach's request, the original thread should appear red and when clicked, show a new message in the thread from the user
#### What are the relevant pivotal tracker stories?
[#163801895](https://www.pivotaltracker.com/story/show/163801895)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
